### ISSUE TITLE
3.6.1 section missing from changelog

### ISSUE THREAD

**Issue by edmorley** [pytest-dev/pytest#3538]
Hi!

There is no v3.6.1 section in the changelog:
https://docs.pytest.org/en/latest/changelog.html
https://github.com/pytest-dev/pytest/blob/master/CHANGELOG.rst

...which means that PRs opened by pyup are missing the release notes, eg:
mozilla/treeherder#3614

Would it be possible to ensure the changelog entries are present before publishing? (Since even if they are added after, it's too late for the PRs that have already been auto-generated).

Many thanks :-)

**Comment by pytestbot** [comment]
[GitMate.io](https://gitmate.io) thinks possibly related issues are https://github.com/pytest-dev/pytest/issues/3536 (3.6.1 fails on "setup.py test"), https://github.com/pytest-dev/pytest/issues/1275 (Separate issues in the CHANGELOG into sections), https://github.com/pytest-dev/pytest/issues/1627 (Drop Python 3.0/3.1/3.2/2.6?), https://github.com/pytest-dev/pytest/issues/2485 (max recursion depth in 3.1.2), and https://github.com/pytest-dev/pytest/issues/722 (Missing tag for xdist release 1.11).

**Comment by nicoddemus** [comment]
Thanks @edmorley for reporting.

What happened is that we took a little longer to merge than usual. I started the process at night by opening the PR, and merged early in the morning by pushing the tag, but got busy with other things and only merged the PR itself several hours later.

> Would it be possible to ensure the changelog entries are present before publishing? (Since even if they are added after, it's too late for the PRs that have already been auto-generated).

With the current process no, because the package is published from the PR... but we will try to reduce the gap between publishing and merging as much as possible in the future.

Thanks again! 👍
